--- Font-TTF-0.14.man/lib/Font/TTF/Manual.pod	Sat Sep 11 03:05:48 1999
+++ Font-TTF-0.14/lib/Font/TTF/Manual.pod	Fri Nov  5 02:11:32 1999
@@ -1,5 +1,5 @@
 
-=head1 TITLE
+=head1 NAME
 
 Manual.pod - Information regarding the whole module set
 
--- Font-TTF-0.14.man/lib/Font/TTF/Maxp.pm	Sat Sep 11 03:05:48 1999
+++ Font-TTF-0.14/lib/Font/TTF/Maxp.pm	Fri Nov  5 02:11:10 1999
@@ -1,6 +1,6 @@
 package Font::TTF::Maxp;
 
-=head1
+=head1 NAME
 
 Font::TTF::Maxp - Maximum Profile table in a font
 
